### PR TITLE
FIXED inspect.getargspec() deprecated in python 3.11

### PR DESCRIPTION
**Pull request by BlackOrder**
Jasmin keep crashing in python 3.11, had to rollback to v3.9 to be stable.

Fix # module 'inspect' has no attribute 'getargspec' 

### Description
Rolled back the image to v3.9 from 3.11 and added the lock cleaning logic from the other image.

### Checks

- [x] My pull request adheres to the code style of this project
- [x] All the tests have passed

### CODE REVIEW

File: docker/Dockerfile.restapi [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM python:3-buster
```
Comment (farirat) [inline docker/Dockerfile.restapi]: The issue with inspect.getargspec() is already fixed in #1085 and will be released on next week, please rollback this change.

File: docker/Dockerfile.restapi.alpine [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM python:3-alpine
```
Comment (farirat) [inline docker/Dockerfile.restapi.alpine]: The issue with inspect.getargspec() is already fixed in #1085 and will be released on next week, please rollback this change.

File: docker/Dockerfile.restapi [hunk 3]
```diff
@@ -1,4 +1,4 @@
-FROM python:3.10-buster
```
Comment (farirat) [inline docker/Dockerfile.restapi]: The issue with inspect.getargspec() is already fixed in #1085 and will be released on next week, please rollback this change.

File: docker/Dockerfile.restapi.alpine [hunk 4]
```diff
@@ -1,4 +1,4 @@
-FROM python:3.10-alpine
```
Comment (farirat) [inline docker/Dockerfile.restapi.alpine]: The issue with inspect.getargspec() is already fixed in #1085 and will be released on next week, please rollback this change.

Comment (farirat) [reply docker/Dockerfile.restapi.alpine]: And it's better to keep same base image as other dockerfiles (python:3-alpine)

File: docker/Dockerfile.restapi (hunk 3 above)
Comment (farirat) [reply docker/Dockerfile.restapi]: And it's better to keep same base image as other dockerfiles (python:3-alpine)

**Review by farirat — CHANGES_REQUESTED** [review]
Thank you for this PR, please take a look to the comments below.

**Comment by BlackOrder** [comment]
Thanks @farirat, i reverted the changes to the python image. but will keep the lock cleaning change. thank you.

**Review by farirat — APPROVED** [review]

**Comment by WilfriedTIANI** [comment]
Good morning all,

Please i face this issue on python 3.10.6
please how to get the working branch

**Comment by farirat** [comment]
hello @WilfriedTIANI 
Releasing is in progress: https://github.com/jookies/jasmin/pull/1095

**Pull request merged**